src/storage/src/hummock/shared_buffer/shared_buffer_batch.rs

```diff
@@ -59,7 +67,7 @@ pub(crate) struct SharedBufferBatchInner {
     /// The epochs of the data in batch, sorted in ascending order (old to new)
     epochs: Vec<HummockEpoch>,
     monotonic_tombstone_events: Vec<MonotonicDeleteEvent>,
-    largest_table_key: Vec<u8>,
+    largest_table_key: Bound<Vec<u8>>,
```

Comment:
We may by the way change the inner type from `Vec<u8>` to `Bytes` to avoid unnecessary memory allocation and copy. So is `smallest_table_key`.

Reply:
> change the inner type from `Vec<u8>` to `Bytes`

fixed